### PR TITLE
Update geoupupdate to 2.2.1

### PR DESCRIPTION
**Pull request by oschwald**

### CODE REVIEW

File: Library/Formula/geoipupdate.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def install
     # See https://github.com/maxmind/geoip-api-c#150
     inreplace "conf/GeoIP.conf.default", "YOUR_USER_ID_HERE", "999999"
     inreplace "conf/GeoIP.conf.default", "YOUR_LICENSE_KEY_HERE", "000000000000"
-    inreplace "conf/GeoIP.conf.default", /^ProductIds .*$/, "ProductIds 506 533"
+    inreplace "conf/GeoIP.conf.default", /^ProductIds .*$/, "ProductIds 506 533 GeoLite2-City GeoLite2-Country"
```
Comment (mikemcquaid) [inline Library/Formula/geoipupdate.rb]: Can you explain this change? Thanks!

Comment (oschwald) [inline Library/Formula/geoipupdate.rb]: This adds the [GeoLite2 databases](http://dev.maxmind.com/geoip/geoip2/geolite2) used by libmaxminddb (and all the newer MaxMind APIs) to the default configuration. 506 and 533 are GeoLite Legacy.

Comment (mikemcquaid) [inline Library/Formula/geoipupdate.rb]: Ok, just trying to understand how that fits into the `2.2.1` bump. Are they installed by defaults and freely usable?

Comment (oschwald) [inline Library/Formula/geoipupdate.rb]: Sorry, I should have made the commit or PR message more specific. They are licensed under the same license as GeoLite Legacy, CC BY-SA 3.0. With this configuration, they will be installed or updated when geoipupdate is run, as will the legacy databases.

Comment (mikemcquaid) [inline Library/Formula/geoipupdate.rb]: Thanks!

**Pull request closed without merging**